src/test/scala/com/microsoft/hyperspace/index/E2EHyperspaceRulesTests.scala

```diff
@@ -407,6 +409,79 @@ class E2EHyperspaceRulesTests extends HyperspaceSuite with SQLHelper {
       sortedRowsWithHyperspaceDisabled.sameElements(getSortedRows(dfAfterHyperspaceDisabled)))
   }
 
+  test("Verify Join Index Rule utilizes indexes correctly after incremental refresh.") {
+    withTempDir { testDir =>
+      val refreshTestLocation = testDir + "/refresh"
+      withSQLConf(IndexConstants.REFRESH_APPEND_ENABLED -> "true") {
+        // Setup. Create Data.
+        val indexConfig = IndexConfig(s"index", Seq("c2"), Seq("c4"))
+        import spark.implicits._
+        SampleData.testData
+          .toDF("c1", "c2", "c3", "c4", "c5")
+          .limit(10)
+          .write
+          .parquet(refreshTestLocation)
+        val df = spark.read.load(refreshTestLocation)
+
+        // Create Index.
+        hyperspace.createIndex(df, indexConfig)
+
+        // Append to Original Data.
```

Comment:
Can we respect casing in all of the comments in this file?

Reply:
thanks @imback82 , added